tiledb/sm/filesystem/azure.cc

```diff
@@ -341,9 +341,9 @@ void Azure::flush_blob(const URI& uri) {
     // Alternatively, we could do nothing and let Azure release the
     // uncommited blocks ~7 days later. We chose to delete the blob
     // as a best-effort operation. We are intentionally ignoring any
-    // errors thrown by `remove_blob`.
+    // errors thrown by `remove_file`.
     try {
-      remove_blob(uri);
+      remove_file(uri);
     } catch (...) {
     }
```

Comment:
Not related to this PR, but on second look deleting the blob to abort an upload is not correct, because we would get data loss if another client has committed an upload to the same blob in the meantime.

I would delete this block now personally, feel free to do it now or later depending on your preference.